src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -2,6 +2,69 @@
 
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
```

Comment:
your method is too huge. Try to shorten it and use additional methods for some actions. Try to avoid code duplication. Also fix code style issues across the file